### DISCUSSION TITLE
Is HTTP/3 stable on .NET 7?

### DISCUSSION THREAD

**Discussion by giggio** [dotnet/runtime#75930] [General] [answered]
On #57600 it says the goal is to stabilize HTTP/3 ("ready for end to end usage").
It is not clear it is stable: the issue was closed, but without much explanation.

I also noticed that on the aspnetcore repo there is still one issue open: https://github.com/dotnet/aspnetcore/issues/36486.

So, will it be ready and out of preview for .NET 7.0 and ASP.NET Core 7?

**Comment by stephentoub** [comment, marked as answer]
> So, will it be ready and out of preview for .NET 7.0 and ASP.NET Core 7?

Yes

The cited ASP.NET issue isn't about it being supported but rather whether you need to opt-in to serving HTTP/3.
